Sources/CorePayments/EligibilityAPI.swift

```diff
@@ -1,52 +0,0 @@
-import Foundation
-
-/// API that return merchant's eligibility for different payment methods: Venmo, PayPal, PayPal Credit, Pay Later & credit card
-class EligibilityAPI {
```

Comment:
Can we also remove `Eligibility.swift`?

Reply:
@KunJeongPark - Actually, this should be it's own PR, the removal of the Eligibility stuff. Especially since we squash and merge PRs, we want to be able to easy revert vault work separately from removing eligibly